oci8.go

```diff
@@ -216,6 +216,28 @@ WrapOCIEnvCreate(ub4 mode, size_t extra) {
   return vvv;
 }
 
+static ret2ptr
+WrapxaoEnv() {
+  ret2ptr vvv = {NULL, NULL, 0};
+
+#ifdef OCI8_ENABLE_XA
+  vvv.ptr = (dvoid *)xaoEnv(NULL);
```

Comment:
You can separate files oci8_xa.go which include this part and build-constraint tag like below.

```go
// +build xa

package oci8

/*
...
static ret2ptr
WrapxaoEnv() {
  ret2ptr vvv = {NULL, NULL, 0};
  vvv.ptr = (dvoid *)xaoEnv(NULL);
  return vvv;
}

static ret2ptr
WrapxaoSvcCtx() {
  ret2ptr vvv = {NULL, NULL, 0};
  vvv.ptr = (dvoid *)xaoSvcCtx(NULL);
  return vvv;
}
```

Reply:
OK will fix.